index.bs

```diff
@@ -257,7 +257,8 @@ Try to construct a table with the columns of the properties of <var>tabularData<
 
 <h4 id="trace" oldids="trace-data,dom-console-trace" method for="console">trace(...<var>data</var>)</h4>
 
-Perform Logger("error", <var>data</var>).
+Performs Logger("log", <var>data</var>) and prints a snapshot of the call-stack from where the method was called.
```

Comment:
> And then modify Printer to talk about console.trace() in addition to %o and %O.

The formatters `%o` and `%O` are discussed in Formatter, not in Printer. Do you mean to add a step between Printer's steps 2 and 3 that would format the opaque trace object?